scripts/Makefile.build

```diff
@@ -135,6 +135,9 @@ quiet_cmd_cc_o_c = CC $(quiet_modtag)  $@
 
 cmd_cc_o_c = $(CC) $(c_flags) -c -o $@ $<
 
+cmd_cxx_o_cpp = $(CXX)  $(cpp_flags) -c -o $@ $<
```

Comment:
I needed to use `$(c_flags)` here to make things work, not `$(cpp_flags)`. In particular, include directories added in `Kbuild` files with `ccflags-y += ...` did not work until I changed this to `$(c_flags)`.

Reply:
Could you try with latest added commit in PR?